script.js

```diff
@@ -1,4 +1,59 @@
+// 1. Generate random object: Scissors, Paper or Stone
+var rollScissorPaperStone = function () {
+  var randomDecimal = Math.random() * 3;
+  var randomInteger = Math.floor(randomDecimal);
+  var signRolled = 0;
+  if (randomInteger == 0) {
+    signRolled = `scissors`;
+  }
+  if (randomInteger == 1) {
+    signRolled = `paper`;
+  }
+  if (randomInteger == 2) {
+    signRolled = `stone`;
+  }
+  return signRolled;
+};
+
+const signs = [`scissors`, `paper`, `stone`];
```

Comment:
Nice trying this out in advance, could have been used for assigning signedRolled in rollScissorsPaper stone function